OWML.Launcher/Program.cs

```diff
@@ -14,7 +14,7 @@ static void Main(string[] args)
         {
             var owmlConfig = GetOwmlConfig();
             var owmlManifest = GetOwmlManifest();
-            var writer = new OutputWriter();
+            var writer = GetWriter(owmlManifest);
```

Comment:
I did it like this for now, but it sounds like a bad idea. I'm thinking the best way to do this is to leave the writer as is (writing to standard output), and then have the mod manager also read from stdout in addition to the socket. This way, if something goes wrong with the socket, we still get some messages. Thoughts?